src/instructlab/model/remove.py

```diff
@@ -0,0 +1,50 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Standard
+from pathlib import Path
+import shutil
+import sys
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.configuration import DEFAULTS
+from instructlab.model.backends.backends import is_model_gguf, is_model_safetensors
+
+
+@click.command(name="remove")
+@clickext.display_params
+@click.option(
+    "--model",
+    required=True,
+    help="Specify the model name to remove(check with `ilab model list` first), \n"
+    "e.g. merlinite-7b-lab-Q4_K_M.gguf or instructlab/merlinite-7b-pt.",
+)
+def remove_model(model: str):
+    """Remove model"""
+    models_dir = Path(DEFAULTS.MODELS_DIR)
+    model_path = models_dir / model
+    # Check if model path exists
+    try:
+        if model_path.exists():
+            if model_path.is_file() and is_model_gguf(model_path):
+                print(f"==> Removing model file: {model}")
+                model_path.unlink()
```

Comment:
I am thinking we probably need a confirm here with a -y or non-interactive mode.